hnn_core/tests/test_rmse.py

```diff
@@ -0,0 +1,31 @@
+import os.path as op
+
+from numpy import loadtxt, c_
```

Comment:
```suggestion
import numpy as np
```

that's the canonical way to import `numpy`